From cfc720d62504a4f8cef7bc5369d1784e25130b61 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Jirman?= <megi@xff.cz>
Date: Fri, 4 Feb 2022 02:26:50 +0100
Subject: [PATCH 372/456] usb: typec-extcon: Don't touch charger proprties

These are owned by usb2 phy driver. Touching them here only introduces
race conditions.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/usb/typec/typec-extcon.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/usb/typec/typec-extcon.c b/drivers/usb/typec/typec-extcon.c
index f1fe5e57fdfc..447aaafca0c9 100644
--- a/drivers/usb/typec/typec-extcon.c
+++ b/drivers/usb/typec/typec-extcon.c
@@ -96,11 +96,6 @@ static int typec_extcon_sync_extcon(struct typec_extcon *tce)
 		typec_extcon_set_cable(tce, EXTCON_USB_HOST, false, prop_ss, prop_or);
 		typec_extcon_set_cable(tce, EXTCON_DISP_DP, false, prop_ss, prop_or);
 
-		extcon_set_state_sync(tce->extcon, EXTCON_CHG_USB_SDP, false);
-		extcon_set_state_sync(tce->extcon, EXTCON_CHG_USB_DCP, false);
-		extcon_set_state_sync(tce->extcon, EXTCON_CHG_USB_CDP, false);
-		extcon_set_state_sync(tce->extcon, EXTCON_CHG_USB_ACA, false);
-
                 goto out_unlock;
 	}
 
-- 
2.34.1

